.github/workflows/magma-build-3rd-party.yml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Magma Build & Publish 3rd Party Dependencies
+on:
+  push:
+    branches:
+      - master
+      - 'v1.*'
+    paths:
+      - third_party/**
+      - .github/workflows/magma-build-3rd-party.yml
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+    branches:
+      - master
+      - 'v1.*'
+    paths:
+      - third_party/**
+      - .github/workflows/magma-build-3rd-party.yml
+  schedule:
+    - cron: 0 4 * * 0
+  workflow_dispatch:
+    inputs:
+      production:
+        description: Is production build?
+        type: boolean
+        default: false
+
+jobs:
+  build_dependencies:
+    runs-on: ubuntu-latest
+    container: ghcr.io/magma/magma/devcontainer:latest
+    env:
+      repository: debian-ci
+
+    strategy:
+      fail-fast: false
+      matrix:
+        distribution: [ focal ]
+        architecture: [ amd64, all ]
```

Comment:
is this now uploading all packages as `amd64` and `all`? I assume we want some packages uploaded as amd64 and some as all.

Reply:
Yes, but I do not see any issue with that. We can well have packages in two places. This does not hurt.
In contrast, I would rather favour this duplication in contrast to the extra complexity of specifically making exceptions for every package. Overall I think it is cheaper, to have this duplication in storage here, compared to the additional effort and complexity otherwise.

Reply:
If someone installs, let's say, folly on arm and the all package is selected by apt then it can not be installed. I think this needs to be separated.

Reply:
Relying on the output of the `build.py` now. Did not find any other place to make the mapping.